code-of-conduct.md

```diff
@@ -72,16 +72,18 @@ Conduct may be permanently removed from the project team.
 
 For incidents occuring in the Kubernetes community, contact the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. You can expect a response within three business days.
 
-For other projects, please contact the CNCF staff via <conduct@cncf.io>. You can expect a response within three business days. 
+For other projects, or for incidents that are project-agnostic or impact multiple CNCF projects, please contact the [CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) via conduct@cncf.io.  Alternatively, you can contact any of the individual members of the [CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) to submit your report. For more detailed instructions on how to submit a report, including how to submit a report anonymously, please see our Incident Resolution Procedures. You can expect a response within three business days.
 
 In matters that require an outside mediator, CNCF has retained Mishi Choudhary (mishi@linux.com). Use of an outside mediator can be requested when reporting or used at CNCF staff's discretion. In general, contacting <conduct@cncf.io> directly is preferred.
 
 
 ## Enforcement 
 
-The Kubernetes project's [Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct) enforces code of conduct issues. For all other projects, the CNCF enforces code of conduct issues.
+The Kubernetes project's [Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct) enforces code of conduct issues.
 
-Both bodies try to resolve incidents without punishment, but may remove people from the project or CNCF communities at their discretion. 
+For all projects that do not have their own Code of Conduct Committee or other Code of Conduct incident responders, and for incidents that are project-agnostic or impact multiple projects, the CNCF Code of Conduct Committee enforces code of conduct issues. For more information, see our Jurisdiction and Escalation Policy
```

Comment:
need link to `Jurisdiction and Escalation Policy`

Reply:
[Jurisdiction and Escalation Policy](https://www.cncf.io/conduct/jurisdiction)